Documentation updates (#358)
syntasso/kratix-docs: docs/_partials/_whats-next.md

alt="Single Kratix Promise"
/>

Platform teams in any form add tremendous value to an organisation. They reduce cognitive load for application teams, which enables those teams to have faster "flow".

> However, reducing cognitive load for application teams generally, in our experience, increases toil for platform team members. The closer a platform team gets to effective enablement of application teams, the higher the likelihood is that, behind the scenes, platform team members are stretched beyond capacity.

Reviewer: Could this be:
```suggestion
However, reducing cognitive load for application teams generally, in our experience, increases toil for platform team members. The closer a platform team gets to effective enablement of application teams, the more likely it is that behind the scenes, platform team members are stretched beyond capacity.
```

> Here is an experience you might have on a platform team as you work to support your application teams who need data storage for their services.

Author: Ah, good feedback! I wasn't happy about the repetition, but I did feel that something felt off after the change. I couldn't put my finger on it, though.

> Here is an experience you might have on a platform team as you work to support your application teams who need data storage for their services.

Author: Reverted my change.
